pkg/nuctl/test/project_test.go

```diff
@@ -537,6 +684,27 @@ func (suite *projectExportImportTestSuite) assertFunctionEventExistenceByFunctio
 	return functionEvent.Meta.Name
 }
 
+func (suite *projectExportImportTestSuite) getExportedProject(projectName string,
+	positionalArgs []string) *command.ImportProjectConfig {
+
+	// reset output buffer for reading the nex output cleanly
+	suite.outputBuffer.Reset()
+
+	// export the project
+	exportProjectPositionalArgs := []string{"export", "project", projectName, "--verbose"}
+	exportProjectPositionalArgs = append(exportProjectPositionalArgs, positionalArgs...)
+	err := suite.RetryExecuteNuctlUntilSuccessful(exportProjectPositionalArgs,
+		nil,
+		false)
+	suite.Require().NoError(err)
+
+	exportedProjectConfig := &command.ImportProjectConfig{}
```

Comment:
If this line is a thing, maybe another indication something in the naming has to change?